specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-08-01/examples/SnapshotsListByResourceGroup.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "snapshot1",
+            "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/snapshots/snapshot1",
+            "type": "Microsoft.ContainerService/Snapshots",
+            "location": "westus",
+            "tags": {
+              "key1": "val1",
+              "key2": "val2"
+            },
+            "systemData": {
+              "createdBy": "user1",
+              "createdByType": "User",
+              "createdAt": "2021-08-09T20:13:23.298420761Z"
+            },
+            "properties": {
+              "creationData": {
+                "sourceResourceId": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/cluster1/agentPools/pool0"
+              },
+              "snapshotType": "Nodepool"
```

Comment:
This is called `AgentPool` elsewhere in the API, right?

For example the URL is `/agentPools`.

Shouldn't we be consistent and use that terminology here too?

Reply:
As we sync-up with PM offline, change "Nodepool" to 'NodePool'